tests/Label/ReadModels/JSON/Repository/AppConfigReadRepositoryDecoratorTest.php

```diff
@@ -130,4 +130,21 @@ public function it_returns_true_if_the_decorator_returns_false_but_the_client_ha
         $canUseLabel = $this->appConfigReadRepositoryDecorator->canUseLabel($userId, $label);
         $this->assertTrue($canUseLabel);
     }
+
+    /**
+     * @test
+     */
+    public function it_returns_true_if_the_decorator_returns_false_but_the_client_has_the_label_in_a_different_case_in_the_config(): void
```

Comment:
It's the `decoratee` which returns false. The `decoratee` is the class that gets a slightly different implementation by adding decoration around it with a `decorator` class.

But I guess it's fine to make the name of this method more compact and just make the case-insensitive part clear.